src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -20,18 +20,7 @@
 import com.google.javascript.jscomp.TypedScope;
 import com.google.javascript.jscomp.TypedVar;
 import com.google.javascript.rhino.Node;
-import com.google.javascript.rhino.jstype.EnumElementType;
-import com.google.javascript.rhino.jstype.EnumType;
-import com.google.javascript.rhino.jstype.FunctionType;
-import com.google.javascript.rhino.jstype.JSType;
-import com.google.javascript.rhino.jstype.NamedType;
-import com.google.javascript.rhino.jstype.NoType;
-import com.google.javascript.rhino.jstype.ObjectType;
-import com.google.javascript.rhino.jstype.ProxyObjectType;
-import com.google.javascript.rhino.jstype.TemplateType;
-import com.google.javascript.rhino.jstype.TemplatizedType;
-import com.google.javascript.rhino.jstype.UnionType;
-import com.google.javascript.rhino.jstype.Visitor;
+import com.google.javascript.rhino.jstype.*;
```

Comment:
you should grab the google style guide
https://docs.google.com/document/d/1GIdeE-sevA8h5SipHoC1r9YPN7MAI0rnm6vMWYyuJZQ/view#heading=h.v8ow5xu1iioa
then IJ will make your imports match the existing code